client/app/telecom/pack/xdsl/modem/wifi/config/pack-xdsl-modem-wifi-config.controller.js

```diff
@@ -1,5 +1,5 @@
 angular.module('managerApp')
-  .controller('XdslModemWifiConfigCtrl', function ($state, $q, $timeout, $stateParams, $translate, TucToast, OvhApiXdsl, TucPackXdslModemMediator) {
+  .controller('XdslModemWifiConfigCtrl', function ($state, $q, $timeout, $stateParams, $translate, TucToast, OvhApiXdsl, TucPackXdslModemMediator, OvhApiXdslModemAvailableWLANChannel) {
```

Comment:
Require a new release of `ovh-api-services`.

Reply:
New release is available: https://github.com/ovh-ux/ovh-api-services/releases/tag/v3.19.0

Reply:
upgrade version